packages/vector_graphics/lib/src/debug.dart

```diff
@@ -2,4 +2,4 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-export '_debug_web.dart' if (dart.library.io) '_debug_io.dart';
```

Comment:
This code was changed for wasm support in https://github.com/flutter/packages/pull/7991; why is it being re-changed here?

Reply:
Because I didn't now this one works too, I always used the inverse with `dart.library.js_interop` to be consistent